include/CppUTest/CppUTestConfig.h

```diff
@@ -176,6 +176,24 @@
 #endif
 #endif
 
+/* Support for "long long" type; can be enabled or disabled externally or by compiler/platform detects. */
+#if !defined(CPPUTEST_USE_LONG_LONG) && !defined(CPPUTEST_NO_LONG_LONG)
+#if defined(_MSC_VER)
+/* "long long" is always supported without warnings on MSVC */
+#define CPPUTEST_USE_LONG_LONG
+#elif defined(HAVE_LONG_LONG_INT)
+/* config.h says the compiler supports it */
```

Comment:
Yup, but this isn't fully working yet because we ought to ship a variant of config.h when it is created. Right now, we don't :(